.devcontainer/Dockerfile

```diff
@@ -6,10 +6,11 @@ FROM mcr.microsoft.com/vscode/devcontainers/javascript-node:0-${VARIANT}
 # On how to use docker from within docker
 
 # Install Docker CE CLI
+ARG BUILDARCH
 RUN apt-get update \
   && apt-get install -y apt-transport-https ca-certificates curl gnupg2 lsb-release \
   && curl -fsSL https://download.docker.com/linux/$(lsb_release -is | tr '[:upper:]' '[:lower:]')/gpg | apt-key add - 2>/dev/null \
-  && echo "deb [arch=amd64] https://download.docker.com/linux/$(lsb_release -is | tr '[:upper:]' '[:lower:]') $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list \
+  && echo "deb [arch=$BUILDARCH] https://download.docker.com/linux/$(lsb_release -is | tr '[:upper:]' '[:lower:]') $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list \
```

Comment:
$BUILDARCH could be replaced with `amd64,arm64`  or `arch=$(dpkg --print-architecture)`